src/nativeapm/nativeapm.ts

```diff
@@ -545,5 +543,12 @@ module ProcessOut {
       };
       document.head.appendChild(markdownScript);
     }
+
+    private loadQrCodesLibrary() {
+      const qrCodeScript = document.createElement("script");
+      qrCodeScript.src =
+        "https://cdnjs.cloudflare.com/ajax/libs/qrcodejs/1.0.0/qrcode.min.js";
```

Comment:
dont we do pinning?

Reply:
yeah, sorry I forgot 🥲 I will upload this lib to checkout-cdn